Makefile

```
@@ -1,7 +1,7 @@
 # Each major sub-repository in this monorepo has their own Makefiles;
 # instead of making an even more compilicated common one, let's delegate to them.
 
-SUBTREES := fendermint ipc ipld-resolver fvm-utils contracts
+SUBTREES := fendermint ipc ipld contracts
```

```suggestion
SUBTREES := fendermint ipc ipld/resolver contracts
```

I haven't seen yet if there is now a `Makefile` under `ipld`, but this results in a `cd <subtree>` and expect `make lint` to work for example.